modules/types/src/any.rs

```diff
@@ -18,6 +21,19 @@ impl Any {
     }
 }
 
+impl Debug for Any {
+    fn fmt(&self, f: &mut Formatter<'_>) -> DebugResult {
+        let mut debug = f.debug_struct("Any");
+        debug.field("type_url", &self.type_url);
+        if self.value.len() > MAX_VALUE_LENGTH_FOR_DEBUG {
+            debug.field("value_size", &self.value.len());
```

Comment:
@yoshidan Instead of hiding the value completely, we can keep a bounded preview:
```rust
debug.field(
    "value",
    &format_args!("{:?} … ({} bytes total)", &self.value[..MAX_VALUE_LENGTH_FOR_DEBUG], self.value.len()),
);
```